hub-js/src/local/mutation/create-type-instance.ts

```diff
@@ -0,0 +1,31 @@
+import { Context } from "./context";
+import {
+  CreateTypeInstanceInput,
+  CreateTypeInstancesInput,
+  TypeInstanceUsesRelationInput,
+} from "../types/type-instance";
+import { createTypeInstances } from "./create-type-instances";
+
+interface createTypeInstancesArgs {
+  in: CreateTypeInstanceInput;
+}
+
+export async function createTypeInstance(
```

Comment:
I don't quite like the fact that we return the alias for the create TI. Maybe we can get rid of it from output? But I believe then we would need a dedicated type for input, without the `alias` field 🤔  That shouldn't be a huge change. WDYT?

Reply:
I removed it 👍 

Personally, I see it as a potential  solution. Finally, we will need to deal with https://github.com/capactio/capact/issues/579. With this approach, we can already introduce sth in between. If no "alias" provided (in this case we will need to add such option to input), we can generate it and store in database. In this way we will have the exact same experience as we have with Actions, if a name is not provided we generate a name for it automatically.

But it's sth that requires more discussion, so I will not do that on this PR as suggested 👍